css-view-transitions-2/Overview.bs

```diff
@@ -299,7 +299,13 @@ Note: as per default behavior, the ''@view-transition'' rule can be nested insid
 
 		: <dfn>auto</dfn>
 		:: The transition will be enabled if the navigation is same-origin, without cross-origin
-			redirects, and is not a {{NavigationType/reload}}.
+			redirects, and is either a {{NavigationType/traverse}} navigation or a navigation that's neither
```

Comment:
Mind writing this something like:

and whose {{NavigationType}} is:
- traverse
- push or replace with {{userNavigationInvolvement}} not equal to browser UI

The sub-points make it easier to understand each case and we want to link to existing concepts in the html spec.

Reply:
Done